tensorflow/core/kernels/quantized_conv_ops.cc

```diff
@@ -492,6 +493,14 @@ class QuantizedConv2DOp : public OpKernel {
     // [ batch, in_rows, in_cols, in_depth ]
     const Tensor& input = context->input(0);
 
+    // validate for non zero dimensions of input.
+    int input_dims = input.shape().dims();
```

Comment:
Nit: input_dims here is the rank of the tensor, that is a more readable name

(don't change now, only if we need to do other changes to the PR)